Source/WebCore/platform/graphics/FloatSize.h

```diff
@@ -143,6 +143,16 @@ class FloatSize {
         return FloatSize(m_height, m_width);
     }
 
+    FloatSize scaledBy(float scale) const
+    {
+        return scaledBy(scale, scale);
+    }
+
+    FloatSize scaledBy(float scaleX, float scaleY) const
+    {
+        return FloatSize(m_width * scaleX, m_height * scaleY);
+    }
+
```

Comment:
Same for these. Can you reuse the existing definitions?

https://github.com/WebKit/WebKit/blob/5bff882b42b8d89caf031389f640e95a79b6a01c/Source/WebCore/platform/graphics/FloatSize.h#L102-L110